benchmark/span_operations_benchmark.cpp

```diff
@@ -71,25 +86,113 @@ class TextMapReader final : public opentracing::TextMapReader {
   const std::vector<std::pair<std::string, std::string>>& key_values_;
 };
 }  // namespace
-
-static std::shared_ptr<opentracing::Tracer> MakeTracer() {
+//
+//------------------------------------------------------------------------------
+// MakeRpcTracer
+//------------------------------------------------------------------------------
+static std::shared_ptr<opentracing::Tracer> MakeRpcTracer() {
   lightstep::LightStepTracerOptions options;
   options.access_token = "abc123";
   options.transporter.reset(new NullTransporter{});
   return lightstep::MakeLightStepTracer(std::move(options));
 }
 
-static void BM_SpanCreation(benchmark::State& state) {
-  auto tracer = MakeTracer();
+//--------------------------------------------------------------------------------------------------
+// MakeStreamTracer
+//--------------------------------------------------------------------------------------------------
+static std::shared_ptr<opentracing::Tracer> MakeStreamTracer() {
+  lightstep::LightStepTracerOptions tracer_options;
+  lightstep::StreamRecorderOptions recorder_options;
+  recorder_options.throw_away_spans = true;
```

Comment:
I see, so the benchmark is comparing the old and new code w/o real network transport in both cases.

Reply:
Yeah, it's only benchmarking the user-facing methods.